content/includes/ngf/installation/deploy-ngf-manifests.md

```diff
@@ -0,0 +1,125 @@
+---
+nd-docs: "DOCS-000"
+files:
+- content/ngf/install/manifests.md
+- content/nginx-one/ngf/add-ngf-manifests.md
+---
+
+{{< call-out "note" >}} By default, NGINX Gateway Fabric is installed in the **nginx-gateway** namespace. You can deploy in another namespace by modifying the manifest files. {{< /call-out >}}
```

Comment:
If we're going to turn a large portion of the installation page into an include, then we should utilize these includes in the existing installation doc to prevent duplication.

Reply:
That's my intent -- see line 4.

I know you were concerned about duplicate content, and I'm trying to address that with includes, ref https://github.com/nginx/documentation/pull/902#discussion_r2277756290

But you're correct that I haven't set up the include in /content/ngf/install/manifests.md yet. To minimize complexity, I'm going to set up a follow-up PR.